src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -93,14 +95,14 @@ private void handleDeletePerson() {
      */
     @FXML
     private void handleNewPerson() {
-        Optional<Person> newPerson = Optional.of(new Person());
+        Optional<ReadablePerson> prevInput = Optional.of(new Person());
```

Comment:
Maybe `lastPersonData` is better. `latest` suggests there is an update or something.